### PR TITLE
search function

### PR DESCRIPTION
**Pull request by Atanna128**
Related tickets: https://edu-redmine.sun-asterisk.vn/issues/28246
What: search tour by name, cost and time
Evidence:
image

### CODE REVIEW

**Review by dangthihoa — APPROVED** [review]

**Review by HieuNT-Sun-Asterisk — APPROVED** [review]

**Comment by Atanna128** [comment]
ready

**Comment by Atanna128** [comment]
ready

File: travel/views.py [hunk 1]
```diff
@@ -209,6 +206,40 @@ def booking_delete(request, pk):
         messages.success(request, 'Delete booking success')
         return HttpResponseRedirect(reverse('booking-history')) 
 
+class TourListView(generic.ListView):
+    model = Tour
+    def get_queryset(self):
+        place_query = self.request.GET.get('place')
+        date_query = self.request.GET.get('duration')
+        cost_query = self.request.GET.get('cost')
+        print(type(date_query))
```
Comment (thaodtd-0665) [inline travel/views.py]: vẫn còn print vậy e

Comment (Atanna128) [reply travel/views.py]: em xoa r a:(

**Comment by Atanna128** [comment]
ready

**Review by thaodtd-0665 — APPROVED** [review]

**Pull request merged**